.graphqlconfig.yaml

```diff
@@ -0,0 +1,42 @@
+projects:
+  users:
+    schemaPath: schemas/users.json
+    includes:
+      - '**/*.users.graphql'
+    extensions:
+      endpoints:
+        production:
+          url: 'https://mc-api.commercetools.com/graphql'
+          headers:
+            X-Graphql-Target: mc
+  menus:
+    schemaPath: schemas/menus.json
+    includes:
+      - '**/*.menus.graphql'
+    extensions:
+      endpoints:
+        production:
+          url: 'https://mc.commercetools.com/api/graphql'
+  settings:
+    schemaPath: schemas/settings.json
+    includes:
+      - '**/*.settings.graphql'
+    extensions:
+      endpoints:
+        production:
+          url: 'https://mc-api.commercetools.com/graphql'
```

Comment:
How would a work flow be btw where the schema is only on stage not on prod?

Reply:
I see two options:
- specify an additional `staging` environment
- pass the URL as an env variable

Given that we probably don’t want to expose the staging domain, we could go with option 2. What do you think?

Reply:
I agree. I would make it an env variable while adding the prod urls into the env template do it's the default. Just wanted to make sure we somehow enable working with this before schema updates hit production.